pkg/prometheus/promcfg.go

```diff
@@ -709,6 +709,9 @@ func (cg *ConfigGenerator) generateProbeConfig(
 	if m.Spec.ProberSpec.Scheme != "" {
 		cfg = append(cfg, yaml.MapItem{Key: "scheme", Value: m.Spec.ProberSpec.Scheme})
 	}
+	if m.Spec.ProberSpec.ProxyURL != "" {
```

Comment:
Could we have a unit test for this?

Reply:
I thought adding a separate method is overkills, so I just added the field in the existing test case: 185a4f3